hack/windows/Prepare-Node.ps1

```diff
@@ -82,21 +90,24 @@ mkdir -force C:\var\log\kubelet
 mkdir -force C:\var\lib\kubelet\etc\kubernetes
 mkdir -force C:\etc\kubernetes\pki
 New-Item -path C:\var\lib\kubelet\etc\kubernetes\pki -type SymbolicLink -value C:\etc\kubernetes\pki\
-
+{{CONTAINER_RUNTIME}}
 $StartKubeletFileContent = '$FileContent = Get-Content -Path "/var/lib/kubelet/kubeadm-flags.env"
-$global:KubeletArgs = $FileContent.Trim("KUBELET_KUBEADM_ARGS=`"")
+$global:KubeletArgs = $FileContent.TrimStart(''KUBELET_KUBEADM_ARGS='').Trim(''"'')
+
+$global:containerRuntime = 
 
-$netId = docker network ls -f name=host --format "{{ .ID }}"
+if ($global:containerRuntime -eq "Docker") {
+    $netId = docker network ls -f name=host --format "{{ .ID }}"
 
-if ($netId.Length -lt 1) {
+    if ($netId.Length -lt 1) {
```

Comment:
Indent executed code of the branch

Reply:
I think this is the change you are referring to ?
    if ($netId.Length -lt 1) {
        docker network create -d nat host
    }